runtime/client_http_request.go

```diff
@@ -168,7 +175,7 @@ func (req *ClientHTTPRequest) WriteJSON(
 
 // Do will send the request out.
 func (req *ClientHTTPRequest) Do() (*ClientHTTPResponse, error) {
-	opName := fmt.Sprintf("%s.%s", req.ClientID, req.MethodName)
+	opName := fmt.Sprintf("%s.%s", req.ServiceName, req.MethodName)
```

Comment:
If we do change the client request struct, then `("%s.%s(%s)", req.ClientID, req.MethodName, req.ThriftServiceMethod)` might be a good alternative.